packages/loot-core/src/shared/environment.ts

```diff
@@ -9,3 +9,10 @@ export function isDevelopmentEnvironment() {
 export function isNonProductionEnvironment() {
   return isPreviewEnvironment() || isDevelopmentEnvironment();
 }
+
+export function isElectron() {
```

Comment:
I think `isBrowser` in `loot-core/src/client/platform.ts` handles this already — is that missing something?

Reply:
Sorry just didnt spot that function existed.